source/plugins/ovs-plugin.rst

```diff
@@ -377,7 +418,7 @@ That would set interfaces to type 'vhostuser' and reference the ports created in
 ::
 
       <interface type='vhostuser'>
-         <source type="unix" path="<OVS_PATH>/<port_name>" mode="client"/>
+         <source type="unix" path="<OVS_PATH>/<port_name>" .../>
```

Comment:
It might be helpful to give a specific example wrt <OVS_PATH>
I don't know if 'people' would know what was expected here, I believe it is the path to the OVS database.

Reply:
Done, thanks